web/book/src/SUMMARY.md

```diff
@@ -103,15 +103,16 @@ General information about the project, tooling and development.
 
 - [Integrations](./project/integrations/README.md)
 
-  - [`prqlc CLI`](./project/integrations/prqlc-cli.md)
-  - [ClickHouse](./project/integrations/clickhouse.md)
   - [Jupyter](./project/integrations/jupyter.md)
+  - [qStudio](./project/integrations/qstudio.md)
+  - [ClickHouse](./project/integrations/clickhouse.md)
   - [DuckDB](./project/integrations/duckdb.md)
   - [Prefect](./project/integrations/prefect.md)
   - [VS Code](./project/integrations/vscode.md)
   - [Rill](./project/integrations/rill.md)
-  - [Syntax highlighting](./project/integrations/syntax-highlighting.md)
   - [PostgreSQL](./project/integrations/postgresql.md)
+  - [`prqlc CLI`](./project/integrations/prqlc-cli.md)
+  - [Syntax highlighting](./project/integrations/syntax-highlighting.md)
```

Comment:
Can we restore the old order and put qStudio under DuckDB? We should have the native integrations higher than those which require add-ons... (same on website)

Reply:
The existing one was good I thought! This adds in qStudio

```diff
[12. Integrations](https://prql-lang.org/book/project/integrations/index.html)

    [12.1. prqlc CLI](https://prql-lang.org/book/project/integrations/prqlc-cli.html)
    [12.2. ClickHouse](https://prql-lang.org/book/project/integrations/clickhouse.html)
    [12.3. Jupyter](https://prql-lang.org/book/project/integrations/jupyter.html)
    [12.4. DuckDB](https://prql-lang.org/book/project/integrations/duckdb.html)
+ qStudio
    [12.5. Prefect](https://prql-lang.org/book/project/integrations/prefect.html)
    [12.6. VS Code](https://prql-lang.org/book/project/integrations/vscode.html)
    [12.7. Rill](https://prql-lang.org/book/project/integrations/rill.html)
-   [12.8. Syntax highlighting](https://prql-lang.org/book/project/integrations/syntax-highlighting.html)
    [12.9. PostgreSQL](https://prql-lang.org/book/project/integrations/postgresql.html)
+   [12.8. Syntax highlighting](https://prql-lang.org/book/project/integrations/syntax-highlighting.html)
```

(prob syntax highlighting should be at the bottom, not required for this PR...)

Reply:
Done. It occurred to me to separate the "external "third-party" tools from the "internal-ish" ones like `prqlc` and the syntax highlighting. They felt like different audiences. But I'm totally OK with this presentation